src/System.Management.Automation/help/MUIFileSearcher.cs

```diff
@@ -133,13 +133,11 @@ private string[] GetFiles(string path, string pattern)
                     result.Add(filePath);
                     break;
                 }
-                // If the input is pattern instead of string, we need to use Regex expression.
-                if (pattern.Contains("*") || pattern.Contains("?"))
+
+                string fileName = Path.GetFileName(filePath);
```

Comment:
@gonhidi - thanks for the extra scrutiny - it is definitely appreciated.

It does seem like there should be consistency between wildcards and non-wildcards. Feel free to open an issue.